INN/Largo
# 1901 - Add in order=ASC on the "ASC" case for series landing page posts

## Changes

This pull request makes the following changes:

- Adds in `$args['order'] = 'ASC';` to the `ASC` case in `series-landing.php`

## Why



For #1901 / https://github.com/INN/umbrella-sfpublicpress/issues/167

## Testing/Questions

Features that this PR affects:

- Series landing page posts order


Questions that need to be answered before merging:


Steps to test this PR:

1. Create a series with a few test posts
2. Create a landing page for said series
3. Set "Post Order" to "Oldest First" and verify the posts are sorted from oldest to newest
4. Try all other post orders and verify they still work as expected
<img width="954" alt="Screen Shot 2021-04-02 at 6 26 49 PM" src="https://user-images.githubusercontent.com/18353636/113458670-0a05a980-93e1-11eb-9759-77ba9036f4fb.png">
